.drone.env

```diff
@@ -3,5 +3,5 @@ CORE_COMMITID=a8f4715fd8cdb6ec371c79e05a4a1fa404618918
 CORE_BRANCH=master
 
 # The test runner source for UI tests
-WEB_COMMITID=06745744336efb9238a57a81e49d8be80162d976
+WEB_COMMITID=05e1db6e71ad3d178e050fd1b4af186703200f84
```

Comment:
This is yesterday's commit id. I guess it is close enough, and only used for grabbing the test code - not for getting the actual web app run-time.

Reply:
true, this is only for the web tests. web assets are fetched from the release tarball. IMO good enough to update the web commitid separately.